clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -301,6 +301,24 @@ public void testInvalidKeystorePassword() throws Exception {
             // Expected exception
         }
     }
+
+    /**
+     * Tests that client connections can be created to a server
+     * if null truststore password is used
+     */
+    @Test
+    public void testNullTruststorePassword() throws Exception {
```

Comment:
I'm not sure how this test works. We are removing the truststore password from the config, but we didn't change the code that creates the truststore not to use a password. What am I missing?